apps/web/src/pages/Profile/ProfilePage/ProfilePage.tsx

```diff
@@ -28,8 +28,8 @@ export const ProfileForm = ({ user }: ProfilePageProps) => {
   const intl = useIntl();
 
   const pageTitle = intl.formatMessage({
-    defaultMessage: "Profile information",
-    id: "gTjLic",
+    defaultMessage: "Personal information",
+    id: "g8Ur9z",
```

Comment:
I'm a bit confused by this change. This page contains a section titles "Personal and contact information" but the new title insinuates that the entire page is "Personal information". Should we also rename that section to differentiate it or try and find a more generic page title? @substrae

Reply:
@esizer 

I'm open to changing the title of the section, though I'm not overly concerned it's confusing as it is.

Reply:
Let's go with "Personal Information"

Reply:
@gobyrne I don't think changing these section names makes sense as then it would be the same as the page title which is very confusing. Can we leave this as is? @esizer, can you confirm that not changing these section titles is a better option than changing these section titles?

## Screenshots

![Screen Shot 2023-10-16 at 14 00 00](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/1909d0a0-c936-4051-bb87-ae9f8bde23f9)
![Screen Shot 2023-10-16 at 14 06 09](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/41cfbffe-88af-4499-88d2-882537f57bbb)

Reply:
Yeah, leave it I guess. @RM-1978 can pick it up again with the full copy review.